### PR TITLE
Document updated setup flow

### LINKED ISSUE #35: Update getting started guide to use personal access key

### PR DESCRIPTION
**Pull request by anthmatic**
Resolves #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,13 @@ Before getting started, you'll need an account that has CMS Hub Enterprise and a
...
-```yaml
-defaultPortal: demo
-portals:
-  - name: demo
-    portalId: <portalId>
-    authType: apikey
-    apiKey: <apiKey>
-```
-
+3. If you already have a `hubspot.config.yml` file set up, run `hs auth` to authenticate with your account. Otherwise, run `hs init` to set up your config file.
```
Comment (gcorne) [inline README.md:11]: I think we should make it clear via a screenshot and other info that they will need to regenerate their access key in order to add custom objects if they already have one.

File: README.md [hunk 2]
```diff
@@ -8,23 +8,13 @@ Before getting started, you'll need an account that has CMS Hub Enterprise and a
...
 4. Run `npm run setup` to create the custom object
-5. Run `npm run upload` to upload the `hub-homes` theme
+5. Run `npm run upload` to upload the `Hub Homes` theme
 6. Add your API key by running `hs secrets add APIKEY <apikey>`
```
Comment (williamspiro) [inline README.md:14]: Could we make this more clear that what you are doing here is setting up the secret for a serverless function?

**Comment by miketalley** [comment]
I changed [my PR](https://github.com/HubSpot/cms-custom-objects-example/pull/38) base branch to `add/document-pak-flow` so we can merge our changes. I added a bunch of links in it to make the process a bit easier.

File: README.md (hunk 2 above)
Comment (miketalley) [reply README.md]: Added

File: README.md (hunk 1 above)
Comment (miketalley) [reply README.md]: Added a screenshot and more info to this

**Comment by anthmatic** [comment]
Thanks @miketalley

**Pull request merged**